src/DDTrace/Integrations/IntegrationsLoader.php

```diff
@@ -94,6 +95,10 @@ public function __construct(array $integrations)
                 '\DDTrace\Integrations\Mysqli\MysqliSandboxedIntegration';
             $this->integrations[PDOSandboxedIntegration::NAME] =
                 '\DDTrace\Integrations\PDO\PDOSandboxedIntegration';
+            // Symfony integration sandboxing is disabled until we support spans auto-closing on
```

Comment:
Do we know specifically why we need auto-closing spans here? I have forgotten if it was ever pointed out.

Reply:
Some users we have interacted with in the past are exiting in controllers.